nncf/quantization/quantizer_setup.py

```diff
@@ -1,20 +1,31 @@
 from collections import Counter
+from typing import Dict
+from typing import List
+from typing import Tuple
+
 from copy import deepcopy
-from typing import List, Tuple, Dict
 
 from nncf.common.utils.logger import logger as nncf_logger
-from nncf.nncf_network import InsertionPoint, InsertionType, NNCFNetwork
-from nncf.common.quantization.structs import QuantizerConfig
+from nncf.nncf_network import InsertionPoint
+from nncf.nncf_network import InsertionType
+from nncf.nncf_network import NNCFNetwork
+from nncf.quantization.layers import QuantizerConfig
 from nncf.tensor_statistics.collectors import ReductionShape
-from nncf.tensor_statistics.statistics import TensorStatistic, MinMaxTensorStatistic
+from nncf.tensor_statistics.statistics import MinMaxTensorStatistic
+from nncf.tensor_statistics.statistics import TensorStatistic
 from nncf.utils import get_scale_shape
 
 QuantizationPointId = int
 
 
 class QuantizationPointBase:
-    def __init__(self, insertion_point: InsertionPoint):
+    def __init__(self, insertion_point: InsertionPoint,
+                 scopes_of_directly_quantized_operators=None, is_adjust_padding_applicable=True):
```

Comment:
`scopes_of_directly_quantized_operators` should be a required parameter - it may be set to an empty list to signify the situation when a quantizer really does not affect any node in the NNCFGraph (such as when an output quantizer is being set up), but otherwise this information should be valid at all times.